### DISCUSSION TITLE
2024-02-02 - Content Management Systems - Chewing the Fat 📈

### DISCUSSION THREAD

**Discussion by GordonBeeming** [SSWConsulting/SSW.Rules.Content#7834] [Chewing the Fat]
Hey SSWers,

Let's talk about Content Management Systems!

1. Have you used a git-based cms?

2. What do you think about TinaCMS?

https://ssw.com.au/rules/git-based-cms-solutions/

**Comment by cvionnet** [comment]
Have no experience with TinaCMS, but the solution looks great.
Nice video Gordon 👍

**Comment by tomek-i** [comment]
>Have you used a git-based cms?

No

>What do you think about TinaCMS?

I don't know enough to form a proper opinion on it. 
I used it twice: 
- the first time was OK and fairly easy. 
- the second time it crashed hard due to sync issues with my account and forks etc. and a whole bunch of steps to fix it up

after that I haven't used it anymore and edited the content on github direclty

**Comment by tiagov8** [comment]
I see a lot of potential in TinaCMS since it gives enough freedom to both designers and developers; while still having a nice UX experience for editors. 

I remember having some login issues at start, but nowadays I am happy to use it anytime I have to. For the SSW.Website there is a lot of room for improvement on the way the content is edited/displayed, but it gets better everyday. E.g. Testimonials management are much better after recent changes from @amankumarrr

**Comment by jakebayliss** [comment]
Have you used a git-based cms
No

What do you think about TinaCMS?
Not sure. The video didn't really show enough of either TinaCMS or the other options for me to have any informed opinion. 

But if we are recommending it and using it for our website then I'm guessing it's pretty good!

**Comment by AlexRothwell** [comment]
> Have you used a git-based cms?

I've only used Netlify so far. I find the concept in general a good thing as managing versions and tracking changes with a more traditional CMS is challenging

> What do you think about TinaCMS?

I've not used it myself but the features it supports sound good

**Comment by bahjatalaadelssw** [comment]
> Have you used a git-based cms?
No

> What do you think about TinaCMS?
Upon reading it looks like the positive functionality suits our needs and has room for improvement

https://ssw.com.au/rules/git-based-cms-solutions/

**Comment by matt-goldman** [comment]
> Have you used a git-based cms?

Yep
    
<br>

>What do you think about TinaCMS?

It's a good start

**Comment by MarcusIrmscherSSW** [comment]
Have you used a git-based cms?
No
What do you think about TinaCMS?
It seems like it will allow for a lot more versatility and creativity

**Comment by chrisschultzssw** [comment]
Have you used a git-based cms?

Yes

> What do you think about TinaCMS?

It can be hard to find that balance between ease-of-use and customisability - looks like TinaCMS has done well there.

**Comment by joshbermanssw** [comment]
1. Have you used a git-based cms? 

No 

2. What do you think about TinaCMS? 

I wasnt sure that the video delved into TinaCMS enough for me to understand it entirely; however the support behind it by SSW seems convincing!

**Comment by leahy268** [comment]
>1. Have you used a git-based cms?
Yes, Netlify

    >2. What do you think about TinaCMS?
Fix some UX issues and the SSO issues and it could be a great product

**Comment by YalalCakirSSW** [comment]
> Have you used a git-based cms?

No

> What do you think about TinaCMS?

I've never worked with CMSs. Based on the information on the video, TinaCMS seems like the best option.

**Comment by UlyssesMaclaren** [comment]
Things required to make this usable by non-devs (IMO)
1. fix the onboarding so Warwick doesn't have to spend 1 hour with me to get it working in the 1st place.
2. remove any mention of "Branches" from the UI (that should all be behind the scenes)
3. when I press save (or maybe publish, or submit for review or whatever), a PR should be autogenerated in the backend without me having to go to GitHub and doit all manually

**Comment by zacharykeeping** [comment]
> 1. Have you used a git-based cms?

I have not.

> 2. What do you think about TinaCMS?

Seems like a good solution that should be easy enough to work with for everyone involved.

**Comment by synnoveisabel** [comment]
> 1. Have you used a git-based cms?
No

> 2. What do you think about TinaCMS?
Don't know enough about CMS's but based on the video TinaCMS seems like a good option

**Comment by wicksipedia** [comment]
> 1. Have you used a git-based cms?

Yes

> 2. What do you think about TinaCMS?

I think it has a lot of potential - I did recommend/implement it for the website in the first place ;)

**Comment by JeanThirion** [comment]
+1 to fixing the PRs not being generally on publish, that's a real issue to me.
Also very very keen to see a Digesting the Fat for this one, super interesting topic :)

> Have you used a git-based cms?

Yes - Tina/Netlify

> What do you think about TinaCMS?

Yes - It's good if we can fix the few issues reported (Auth + auto-PRs)

**Comment by jimmy1992abc** [comment]
> Have you used a git-based CMS?

No

> What do you think about TinaCMS?

We can always try if the cost and risk are low. Anything that is potentially able to transfer unused resources into $$ is worth trying.

I am also thinking about why we couldn't allocate people from internal projects to something like building a game to sell on Steam.

- We have the resources and ability
- Won't affect our client work and potential income
- We got game lovers, try something similar to "blood on the clocktower" first.

**Comment by isaaclombardssw** [comment]
> 1.   Have you used a git-based cms?

No.

 > 2. What do you think about TinaCMS?

Similar to others, hard for me to have a specific opinion. Based on the video it seems like a good fit for some scenarios where the ease of use by non-devs is important.

**Comment by TobyChurches** [comment]
>Have you used a git-based cms?

No

>What do you think about TinaCMS?

From what I have seen it sounds like a neat solution. A great tool for lowering the barrier to entry for content modifications on Client solutions.

**Comment by bettybondoc** [comment]
> Have you used a git-based cms?

Yes - Netlify/TinaCMS


> What do you think about TinaCMS?

I've made some content updates using TinaCMS and found it straightforward. I appreciate being able to create branches and having a live preview so I can safely play with content before making a pull request. I haven't done anything beyond minor copy changes so I can't say much else at this point.

If anything, I used Netlify once and never again because I felt it's easier to just edit straight from GitHub. But I didn't have such strong feelings when I used TinaCMS so I take it as a good sign.

**Comment by danielmackay** [comment]
> 1. Have you used a git-based cms?

No

> 2. What do you think about TinaCMS?

Seems ok.  I'd need to test it out to form a stronger opinion.

**Comment by sethdaily** [comment]
I agree with @leahy268:
> Fix some UX issues and the SSO issues and it could be a great product

It's a worthy investment if we can commit resources to make it great - by dropping other projects

**Comment by vladislav-kir** [comment]
> Have you used a git-based cms?

Yes, I used TinaCMS for SSW Website.

> What do you think about TinaCMS?

It seems promising and worked quite well for Website in my experience.

**Comment by tombui99** [comment]
> Have you used a git-based cms?

Yes - I have spent a bit of time working and using it on SSW Website

> What do you think about TinaCMS?

It seems to be a very easy and straightforward solution which I think most devs would prefer especially ones who are familiar with using GitHub. It can be a bit of a learning curve for non-devs when it comes to managing branches and PR on GitHub.

**Comment by Levijj22** [comment]
> Have you used a git-based cms?
No

> What do you think about TinaCMS?
I like it! Although I would make some changes, i.e. GitHub-less interface, fix auth issues, make it more non-dev friendly. 
Lets do it!

**Comment by jeoffreyfischer** [comment]
I've been using TinaCMS for a couple months now and I think it's a great tool. The user interface is quite intuitive, even though the code behind can be tricky at first.

**Comment by miichaelsmedley** [comment]
> Have you used a git-based cms?
No I haven't

Id like to see a true cost benefit analyses - add all of the variables including cost and view that for a decision.

**Comment by ricksu978** [comment]
> 1. Have you used a git-based cms?
No.

> 2. What do you think about TinaCMS?
I haven't used TinaCMS extensively. At first glance, it looks promising, but I need to explore more use cases to gain a deeper understanding.

**Comment by JudyMao99** [comment]
> 1. Have you used a git-based cms?
> No

> 2. What do you think about TinaCMS?
> As a developer, I think it would be a handy tool to provide efficiency but more exploration is needed on it.

**Comment by ozair-ashfaque** [comment]
> 1. Have you used a git-based cms?

No

> 2. What do you think about TinaCMS?

After watching Gordon's video and reading the rule about TinaCMS, it could be a potential alternative.

**Comment by steven0x51** [comment]
1. Have you used a git-based cms?
No, haven't tried a cms yet

2.What do you think about TinaCMS?
The solution looks simple and nice

**Comment by amankumarrr** [comment]
> Have you used a git-based cms?  
> What do you think about TinaCMS?


Yes, I have been using it for a while, developing templates out of it, integrating it with React, and migrating old websites through it has been a really smooth experience. For the end user, it offers visual editing options, providing a unique and intuitive experience.

**Comment by AntPolkanov** [comment]
> Have you used a git-based cms?

No, I haven't

> What do you think about TinaCMS?

It looks nice. I also like that it has 10k+ stars on GitHub

**Comment by KostaMadorsky** [comment]
> 1. Have you used a git-based cms?

No

> 2. What do you think about TinaCMS?

Looks interesting; need more hands-on experience to comment, though.

**Comment by tkapa** [comment]
Have you used a git-based cms?
Yes - And previously they've all felt quite Jank

What do you think about TinaCMS?
It's quite cool and easy to use.

**Comment by Geordie88** [comment]
As a non dev user, it's a really smooth experience... When I don't get any errors. But when I do its often overly technical (need to call either Aman or Brady) for what I should do. given that most of the fixes have been fairly simple having either custom errors with a to do or a more readily available wiki with troubleshooting solutions would be good :).

**Comment by kikibianc** [comment]
> Have you used a git-based cms?

No

> What do you think about TinaCMS?

Looks good in the video but I never used it myself.

**Comment by AshAnil** [comment]
> Have you used a git-based cms?
No

> What do you think about TinaCMS?
Looks good and is easy to edit by a non-developer too. Never used it before.

**Comment by MattParkerDev** [comment]
TinaCMS seems cool, but I haven't used it very much to form a solid opinion.

**Comment by jaydenalchin** [comment]
Onboarding and getting set up was a bit painful, but after that my experience with TinaCMS has been very smooth - infinitely easier to update website content than it once was

**Comment by Marxoz** [comment]
> 1.Have you used a git-based cms?
No I haven't

   > 2.What do you think about TinaCMS?
It sounds like a good solution. Especially the custom components. But will have to give air a proper test drive on a real client before I can say more

**Comment by KahaMason** [comment]
> Have you used a git-based cms?

Yes

> What do you think about TinaCMS?

From Gordon's video and the little play around I had with it, it definitely looks promising.

**Comment by Dhruv-0987** [comment]
1. Have you used a git-based cms?

Been using the decapCMS for rules, its good but goes into async issues sometimes, I usually always use github.

What do you think about TinaCMS?

Its a good start because making changes from Github is really clunky, also the reusable component feature is really cool.

**Comment by SamBWagner** [comment]
Super exciting stuff. I'm keen to work on/with TinaCMS

**Comment by PennyWalker** [comment]
I think it's great when it works, I've had some huge auth issues and have been locked out a few times, but the functionality is nice once you're in. It would be a really big investment to move it to our stack, but it would be a nice real-world project for people on the bench to work on.

**Comment by drwharris** [comment]
Never used TinaCMS but from what I can see, looks like a nice toolset. I am unsure of the use case we want to target its use for and therefore what it would compete against

**Comment by JackDevAU** [comment]
Good video @GordonBeeming !! 

> Have you used a git-based cms?

Yes - quite a lot of them 😅 they all suffer from the same issues. 

> What do you think about TinaCMS?

Seems good, docs seem a bit all over the place. But I'm sure they must be better than I believe them to be as people are loving it. The editor is super clean. But will need to try it out and dev with it a bit more :)

**Comment by stephanfakossw** [comment]
Have you used a git-based cms?
No.

What do you think about TinaCMS?
It's seems to have a future as an open source product.

**Comment by jerryluossw** [comment]
> Have you used a git-based cms?

No.

> What do you think about TinaCMS?

It depends on how many clients we will have? we have 3 types of clients:
1. Clients who willing to put their documentation on Github and they want a CMS to manage it, those are our target clients, the question is: how many?
2. Clients who willing to put their documentation on Github and they have the ability and willingness to edit markdown files directly.
3. Clients who don't want to put their documentation on Github

**Comment by Aibono1225** [comment]
> 1. Have you used a git-based cms?

Yes, I have used Decap CMS

> 2. What do you think about TinaCMS?

With all the comparisons mentioned in the video, TinaCMS looks cool!

**Comment by sylhuang** [comment]
> Have you used a git-based cms?

Yes, Netlify CMS

> What do you think about TinaCMS?

Have no experience but the demo seems to have some great potential

**Comment by hveraus** [comment]
> Have you used a git-based cms?

Yes

> What do you think about TinaCMS?

It's barely usable in China. constantly blocking me from the AWS authentication. If I turn on VPN, the SSW authentication will block me from signing in from a different location.

**Comment by AttackOnMorty** [comment]
> Have you used a git-based cms?

No

> What do you think about TinaCMS?

Looks good. Real-time visual editing is awesome.

**Comment by KristenHu** [comment]
> 1. Have you used a git-based cms?

Yes.
> 2. What do you think about TinaCMS?

A good solution, as long as those small issues like UX issues in TinaCMS can be fixed.

**Comment by kenzengssw** [comment]
> Have you used a git-based cms?

No

> What do you think about TinaCMS?

Looks cool, will definitely give a try.

**Comment by Freego1783** [comment]
> Have you used a git-based cms?

No

> What do you think about TinaCMS?

Looks promising! I may give it a try

**Comment by jimmidier** [comment]
> Have you used a git-based cms?

Yes, Netlify CMS
> What do you think about TinaCMS?

Awesome tool, nice integration with NextJs. It had some issues regarding system compatibility on the server side but it's still a go-to CMS with lots of great designs

**Comment by JackDuan123** [comment]
I like Tina CMS because I just simply like open source

**Comment by BrookJeynes** [comment]
> Have you used a git-based cms?

TinaCMS has been my only experience with git-based CMS

> What do you think about TinaCMS?

I believe it has potential

**Comment by christoment** [comment]
> 1. Have you used a git-based cms?

Yes

> 2. What do you think about TinaCMS?

It's good for git-based CMS.

**Comment by camillars** [comment]
> Have you used a git-based cms?

Yes, TinaCMS.

> What do you think about TinaCMS?

Simple and easy to use despite the login issues. However, as a marketer, you would often need someone technical to ask questions or to get support from (e.g. when compared to Wordpress where you can easily find answers online to fix issues).

**Comment by william-liebenberg** [comment]
> What do you think about TinaCMS?

TinaCMS is good. It has some rough edges around onboarding and authentication and missing features (liek in-client PR UI). It's best features definitely shine brightly (git-based, branching, MDX... and GitHub contributions).

**Comment by JackDevAU** [comment]
Good video @GordonBeeming!

> Have you used a git-based cms?

Yes - quite a lot of them 😅 they all suffer from the same issues.

> What do you think about TinaCMS?

Seems good, docs seem a bit all over the place. But I'm sure they must be better than I believe them to be as people are loving it. The editor is super clean. But will need to try it out and dev with it a bit more :)

**Reply by bradystroud** [reply]
> they all suffer from the same issues

Interested if you have a list and if its similar to https://github.com/SSWConsulting/SSW.Rules/issues/625

**Comment by jernejk** [comment]
> 1. Have you used a git-based CMS?
Some, here and there, but no idea what technology they were. This might not count but I did use and worked with TinyMCE when PHP forums were popular.

Not only are they still alive (I remember them from 2004!), but they seemed to add some AI to it as well!
https://www.tiny.cloud/

> 2. What do you think about TinaCMS?
I heard good things about... No personal experience. 😅

**Comment by tino-liu** [comment]
> Have you used a git-based cms?

No

> What do you think about TinaCMS?

I've used Sanity before. It seems similar. Willing to give TinaCMS a try.

**Comment by bradystroud** [comment]
Tina is the best git based CMS i've touched, but still suffers from the issues mentioned here 
https://github.com/SSWConsulting/SSW.Rules/issues/625

**Comment by babakamyljanovssw** [comment]
Great video!

> 1. Have you used a git-based cms?
No

> 2. What do you think about TinaCMS?
It looks promising, towards to using it and get an experience on CMS

**Comment by jackreimers** [comment]
> Have you used a git-based cms?  

Yes, Tina and Netlify.

> What do you think about TinaCMS?

I think it has a lot of potential but until it can be used without having to interact with Git I don't think I could recommend it for any client or team that isn't solely comprised of developers.

**Comment by bradystroud** [comment]
Tina is the best git based CMS i've touched, but still suffers from the issues mentioned here
https://github.com/SSWConsulting/SSW.Rules/issues/625

**Comment by sethdaily** [comment]
The UI for editing is getting better and better. Soon it will be the best 👑

**Comment by jeoffreyfischer** [comment]
> What's the most important thing about a CMS for you?

That the UX is intuitive

**Comment by bahjatalaadelssw** [comment]
>What's the most important thing about a CMS for you?
Live changes to updates.

**Comment by joshbermanssw** [comment]
> What's the most important thing about a CMS for you? 

Couple things that set Tina as a CMS apart from some of the big players: editing all in one browser, auto sync with GitHub instead of a database and contextual editing

**Comment by isaaclombardssw** [comment]
> What's the most important thing about a CMS for you?

When you have the time to warrant it, you really have a lot of freedom with some of these tools since you can retain control over the code (if they're headless) and also the data (from the git integration).

**Reply by stephanfakossw** [reply]
> What's the most important thing about a CMS for you?

N/a - not a developer

**Comment by RobJT-wq** [comment]
> What's the most important thing about a CMS for you?
I think ease of use is the most important thing for any tool

**Comment by ricksu978** [comment]
The getting started experience has improved.
I ran into error a few months ago, tried again today and it went smooth!

**Comment by Lewkans** [comment]
> What's the most important thing about a CMS for you?

The ease of use and documentation quality for a good experience when starting out

**Comment by bettybondoc** [comment]
> What's the most important thing about a CMS for you?

The most important thing about a CMS for me is that it’s intuitive and feels like a well-designed playground—not a prison cell. It gives me the freedom to be creative and customize as needed, but with enough structure and guidance to keep everything organised and running smoothly.

**Comment by Ben0189** [comment]
If i am a user of CMS, user experience is most important for me

1. How easy i can find the content i want to edit
2. How easy i can make the changes and save 
3. How quickly i can see the changes on lives

**Comment by chrisschultzssw** [comment]
> What's the most important thing about a CMS for you?

Ease of use - I want a CMS to make things easier, not harder!

**Comment by leahy268** [comment]
The startup experience has improved out of sight in the last few months.

**Comment by bradystroud** [comment]
I want to be able to edit content without thinking about how to edit content

**Comment by JackDevAU** [comment]
> What's the most important thing about a CMS for you?

100% the docs - We've updated a lot of it but still a ways to go

**Comment by Tanya-Leahy** [comment]
>Have you used a git-based cms

I have used Tina for creating the Microservices listings for the website

>What do you think about TinaCMS?

I think it is easy to use and make edits in. The software is improving all the time.

**Comment by TobyChurches** [comment]
> Have you used a git-based cms?

No

> What do you think about TinaCMS?

I would want to use it more to get a better Idea but from first glace, I really like it!

**Comment by tkapa** [comment]
TinaCMS is really nice for when you just want to have your stuff working :)

**Comment by jimmidier** [comment]
> Have you used a git-based cms?

Not before TinaCms

> What do you think about TinaCMS?

Very easy to use for editors. Feature rich for developers as well,

**Comment by KailinYuan** [comment]
Have you used a git-based cms?
No
What do you think about TinaCMS?
I haven't used TinaCMS before, but based on the video introduction, it appears to offer greater flexibility and innovation.

**Comment by jerryluossw** [comment]
> Have you used a git-based cms?

No.

> What do you think about TinaCMS?

I've never used it, but according to the videos and the documentations, it's good.
In addtion, our client like it and they may migrate more sites to TinaCMS.

**Comment by thom-wang** [comment]
> Have you used a git-based cms?

No.

> What do you think about TinaCMS?

I think it has the most freindly UI than other content editor.

**Reply by tino-liu** [reply]
> Have you used a git-based cms?

No

> What do you think about TinaCMS?

It is very user friendly when it comes to editing. The visual editing functionality is god-send.

**Comment by amankumarrr** [comment]
I really like TinaCMS as a developer, especially for building editable components that empower the marketing team to manage content independently. However, I believe there’s room to make it more user-friendly and less developer-centric, which would further streamline content editing for non-technical users as well.

**Reply by sam-s-maher** [reply]
> Have you used a git-based cms?

No

> What do you think about TinaCMS?

Looks like it's a great product for hitting cross-functional teams/users/punters - e.g. Developers who want a bit more control than your Wordpress' and Squarespace's, but still want the ease of design of a site builder.

I also love Markdown, so that's a big plus.

**Comment by ncn-ssw** [comment]
The most important thing about a CMS for me is that it provides an intuitive, predictable experience when I change content.

**Comment by awaltos** [comment]
The most important thing about a CMS for me is that it is easy to use for content creators (non-technical). Underlying tech stack and dev experience is a nice to have (i am a developer), but I think the key to a CMS being successful is non-technical people who have to use it are having a good time by easily understanding what content is published and how to make changes.

**Comment by babakamyljanovssw** [comment]
The most important thing about CMS for me is to have visual editing so I can see the changes immediately and save them easily.

**Comment by Aibono1225** [comment]
> Have you used a git-based cms?

No.

> What do you think about TinaCMS?

I've used it a few times, and the experience has been great. The visual editing feature is very impressive to me.

**Comment by MattParkerDev** [comment]
>Have you used a git-based cms?

Yes

> What do you think about TinaCMS?

Looks like a great solution.

**Reply by Marxoz** [reply]
Have you used a git-based cms?

Yes

What do you think about TinaCMS?

Easy to get running. The live editor is really cool
